dll/shellext/netshell/lang/ro-RO.rc

```diff
@@ -139,7 +145,7 @@ BEGIN
     IDS_SHV_COLUMN_TYPE "Tipe"
     IDS_SHV_COLUMN_STATE "Stare"
     IDS_SHV_COLUMN_DEVNAME "Nume dispozitiv"
-    IDS_SHV_COLUMN_PHONE "Tel. sau adresa gazdei"
+    IDS_SHV_COLUMN_PHONE "Număr de telefon sau adresa gazdei"
```

Comment:
This is a column header in a listview. Using such a long string might look weird, as it steals a lot of space from other columns. Please attach a screenshot (at least), and if it looks poor, then please impove on that.